templates/usr/local/bin/rsync_debian_mirror.j2

```diff
@@ -7,22 +7,24 @@
 # uncomment to enable bash debugging
 #set -x
 
-tmp_path="{{ repo_mirror_tmp_path }}"
-base_path="{{ repo_mirror_base_path }}/{{ item.name }}"
-log_file="{{ repo_mirror_log_path }}/{{ item.name }}/repo_{{ item.name }}.log"
-status_file="{{ repo_mirror_base_path }}/{{ item.name }}.json"
+repo_name={{ item.name | quote }}
+tmp_path={{ repo_mirror_tmp_path | quote }}
+base_path={{ repo_mirror_base_path | quote }}/$repo_name
+log_file={{ repo_mirror_log_path | quote }}/$repo_name/repo_$repo_name.log
+status_file=$base_path.json
+owner={{ repo_mirror_user | quote }}
```

Comment:
I haven't fully understood the logic behind the grouping of the variables, so for now I've simply added the declaration here. Please let me know if this should better go elsewhere.

Reply:
Same for the `$owner` declaration in the other 3 scripts.

Reply:
I'm not 100% sure but wouldn't 
```
log_file={{ repo_mirror_log_path | quote }}/$repo_name
```

result in the following script line (notice the quoting)? 

```
log_file="/path/to/log"/repo_name
```

Reply:
```
log_file="/path/to/log"/$repo_name
```
(note the `$`), but yes. But it would be valid, the quotes are interpreted and stripped by the shell (e.g. something like `"hello"world'uieeeeeeee!'` is all ultimately reduced to simply `helloworlduieeeeeeee!`).